pkg/api/handlers/compat/secrets.go

```diff
@@ -25,12 +25,14 @@ func ListSecrets(w http.ResponseWriter, r *http.Request) {
 	}{}
 
 	if err := decoder.Decode(&query, r.URL.Query()); err != nil {
-		utils.Error(w, http.StatusText(http.StatusBadRequest), http.StatusBadRequest,
+		utils.Error(w, "Something went wrong.", http.StatusBadRequest,
```

Comment:
should not change this line

Reply:
The idea was to make it more similar to other endpoints ... I was under the impression that the `"Something went wrong.`-style is the most common. But trying to prove my point I found quite some counter examples (see below). :shrug: 
I'll revert this then.

`"Something went wrong.`-style examples:
- https://github.com/containers/podman/blob/master/pkg/api/handlers/compat/containers.go#L99
- https://github.com/containers/podman/blob/master/pkg/api/handlers/compat/containers.go#L166
- https://github.com/containers/podman/blob/master/pkg/api/handlers/compat/containers.go#L194
- https://github.com/containers/podman/blob/master/pkg/api/handlers/compat/containers_logs.go#L38
- https://github.com/containers/podman/blob/master/pkg/api/handlers/compat/containers_prune.go#L24
- https://github.com/containers/podman/blob/master/pkg/api/handlers/compat/images_remove.go#L24
- https://github.com/containers/podman/blob/master/pkg/api/handlers/compat/images_search.go#L33
- https://github.com/containers/podman/blob/master/pkg/api/handlers/compat/networks.go#L221
- https://github.com/containers/podman/blob/master/pkg/api/handlers/compat/networks.go#L300
- https://github.com/containers/podman/blob/master/pkg/api/handlers/compat/networks.go#L340
- https://github.com/containers/podman/blob/master/pkg/api/handlers/compat/networks.go#L371
- https://github.com/containers/podman/blob/master/pkg/api/handlers/compat/volumes.go#L100
- https://github.com/containers/podman/blob/master/pkg/api/handlers/compat/volumes.go#L277

`http.StatusText(http.StatusBadRequest)`-style examples:
- https://github.com/containers/podman/blob/master/pkg/api/handlers/compat/containers.go#L43
- https://github.com/containers/podman/blob/master/pkg/api/handlers/compat/containers_top.go#L28
- https://github.com/containers/podman/blob/master/pkg/api/handlers/compat/containers_stop.go#L31
- https://github.com/containers/podman/blob/master/pkg/api/handlers/compat/containers_restart.go#L31
- https://github.com/containers/podman/blob/master/pkg/api/handlers/compat/containers_create.go#L28
- https://github.com/containers/podman/blob/master/pkg/api/handlers/compat/volumes.go#L34
- https://github.com/containers/podman/blob/master/pkg/api/handlers/compat/volumes.go#L93

totally different style:
- https://github.com/containers/podman/blob/master/pkg/api/handlers/compat/containers_start.go#L22
- https://github.com/containers/podman/blob/master/pkg/api/handlers/compat/containers_attach.go#L30
- https://github.com/containers/podman/blob/master/pkg/api/handlers/compat/events.go#L87
- https://github.com/containers/podman/blob/master/pkg/api/handlers/compat/containers_archive.go#L39